__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require("../app/app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("GET:200 sends an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+});
+
+describe("GET /api/tpics", () => {
+  test("GET:404 Not Found on endpoint that doesn't exist", () => {
+    return request(app)
+      .get("/api/tpics")
+      .expect(404)
+      .then((body) => {
+        expect(body.res.statusMessage).toBe("Not Found");
+      });
+  });
```

Comment:
This is a valid test, however, it isn't specific to `/api/topics` This would be a test for any endpoint that your server doesn't handle. So `/api/tpics` is the same as `/api/banana`